docs/metastores.md

```diff
@@ -0,0 +1,98 @@
+    <class>org.apache.polaris.core.persistence.models.ModelEntity</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityActive</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityChangeTracking</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityDropped</class>
+    <class>org.apache.polaris.core.persistence.models.ModelGrantRecord</class>
+    <class>org.apache.polaris.core.persistence.models.ModelPrincipalSecrets</class>
+    <class>org.apache.polaris.core.persistence.models.ModelSequenceId</class>
+    <shared-cache-mode>NONE</shared-cache-mode>
+    <properties>
+      <property name="jakarta.persistence.jdbc.url"
+        value="jdbc:h2:file:tmp/polaris_test/filedb_{realm}"/>
+      <property name="jakarta.persistence.jdbc.user" value="sa"/>
+      <property name="jakarta.persistence.jdbc.password" value=""/>
+      <property name="jakarta.persistence.schema-generation.database.action" value="create"/>
+    </properties>
+  </persistence-unit>
+```
+
+Build with H2 dependency and run Polaris service:
+> polaris> ./gradlew --no-daemon --info -PeclipseLink=true -PeclipseLinkDeps=com.h2database:h2:2.3.232 clean shadowJar
+> polaris> java -jar  polaris-service/build/libs/polaris-service-*.jar server ./polaris-server.yml
+
+### Postgres
+
+The following shows a sample configuration for Postgres database.
+
+```angular2html
+  <persistence-unit name="polaris" transaction-type="RESOURCE_LOCAL">
+  <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+  <class>org.apache.polaris.core.persistence.models.ModelEntity</class>
+  <class>org.apache.polaris.core.persistence.models.ModelEntityActive</class>
+  <class>org.apache.polaris.core.persistence.models.ModelEntityChangeTracking</class>
+  <class>org.apache.polaris.core.persistence.models.ModelEntityDropped</class>
+  <class>org.apache.polaris.core.persistence.models.ModelGrantRecord</class>
+  <class>org.apache.polaris.core.persistence.models.ModelPrincipalSecrets</class>
+  <class>org.apache.polaris.core.persistence.models.ModelSequenceId</class>
+  <shared-cache-mode>NONE</shared-cache-mode>
+  <properties>
+    <property name="jakarta.persistence.jdbc.url"
+              value="jdbc:postgresql://localhost:5432/{realm}"/>
+    <property name="jakarta.persistence.jdbc.user" value="postgres"/>
+    <property name="jakarta.persistence.jdbc.password" value="postgres"/>
+    <property name="jakarta.persistence.schema-generation.database.action" value="create"/>
+    <property name="eclipselink.persistence-context.flush-mode" value="auto"/>
+  </properties>
+</persistence-unit>
+```
+Build with Postgres dependency and run Polaris service:
+> polaris> ./gradlew --no-daemon --info -PeclipseLink=true -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4 clean shadowJar
+> polaris> java -jar  polaris-service/build/libs/polaris-service-*.jar server ./polaris-server.yml
```

Comment:
NIT: add one more block quotes (">") here so they will show in two lines instead of one line